src/lib/app-helper.js

```diff
@@ -51,8 +51,37 @@ async function runPackageScript (scriptName, dir, cmdArgs = []) {
   aioLogger.debug(`running npm run-script ${scriptName} in dir: ${dir}`)
   const pkg = await fs.readJSON(path.join(dir, 'package.json'))
   if (pkg && pkg.scripts && pkg.scripts[scriptName]) {
-    return execa('npm', ['run-script', scriptName].concat(cmdArgs), { cwd: dir, stdio: 'inherit' })
-  } else {
+
+    let termListeners = process.listeners('SIGINT')
+    process.removeAllListeners('SIGINT')
+    process.on('SIGINT', () => {
+      if(!childPro.killed) {
+        childPro.kill()
```

Comment:
shouldn't sigint also kill the main process ? seems like the user would need to CTRL +C two times